.travis.yml

```diff
@@ -19,8 +19,7 @@ cache:
   - $HOME/.gradle/wrapper/
 after_success:
 - bash <(curl -s https://codecov.io/bash)
-- ./gradlew clean -Dbuild.version=$TRAVIS_TAG gitPublishPush
-- ./travis-build.sh
+- ./travis-publish.sh || travis_terminate 1
```

Comment:
@andrii-kovalenko-ct what happens here if you don't add the `travis_terminate 1`

Reply:
if `travis-publish.sh` finishes with an error (e.g. exit code ≠ 0) - the build will still be accounted as success. For this reason we should use this construction to fail the build if script is failed